website/src/data/users.tsx

```diff
@@ -602,6 +602,14 @@ const Users: User[] = [
     source: 'https://github.com/eta-dev/eta-docs',
     tags: ['opensource'],
   },
+  {
+    title: 'EverShop',
+    description: 'An open-source e-commerce platform with NodeJS and ReactJS',
+    preview: require('./showcase/evershop.png'),
+    website: 'https://evershop.io/',
+    source: 'https://github.com/evershopcommerce/evershop',
```

Comment:
This does not seem to contain your website's source.

Reply:
Dear @Josh-Cena,

The document site source code is in private repo. The one in my commit is the product repo. Can I just remove the ‘source’ from my commit?

Thanks

Reply:
Ah, sure. Just remove "source" and the "opensource" tag.

Reply:
Dear @Josh-Cena ,

I decided to public the repo and keep the 'source' (updated).

Please review again. Thanks for your approval